modules/core/src/model/BinarySearchTree.ts

```diff
@@ -345,4 +351,59 @@ export class BinarySearchTree extends Cloneable<BinarySearchTree> {
 
   }
 
+  /**
+   * Build Binary Search Tree from the ordered number array
```

Comment:
Should probably also add comment here that this way of creating a bst creates one with a depth of logn.

Reply:
done